src/components/biggive-hero-image/biggive-hero-image.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable prettier/prettier */
```

Comment:
@tomwpaterson-shape Why are turning off style rules as opposed to using the auto lint fix commands?